libs/test-utils/src/expect_print.ts

```diff
@@ -0,0 +1,114 @@
+import { render, RenderResult } from '@testing-library/react';
+import { Optional, Printer, PrintOptions } from '@votingworks/types';
+import { advanceTimersAndPromises } from './advance_timers';
+import { assert } from './assert';
+
+export const fakePrintElement = jest.fn(
+  (_element: JSX.Element, printer: Printer, printOptions: PrintOptions) => {
+    return printer.print(printOptions);
+  }
+);
+
+export const fakePrintElementWhenReady = jest.fn(
+  async (
+    renderElement: (onReadyToPrint: () => void) => JSX.Element,
+    printer: Printer,
+    printOptions: PrintOptions
+  ) => {
+    return printer.print(printOptions);
```

Comment:
Explanation - we don't end up using then because `resetMocks: true` is set in our jest config files, so their implementation is cleared before any test. We might want to turn that off so our `expect(printFn).toHaveBeenCalledTimes(...)` don't get off as we blend testing approaches for the time being. But if the `expectPrint` approach works, we should't have to check calls to `print`. We are trusting (with testing) that the new print utilities are calling `printer.print` at the appropriate time.